src/components/layout/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactNode } from "react";
+
+import Footer from "./footer";
+import Navbar from "./navbar";
+
+interface LayoutProps {
+  children: ReactNode;
+}
+
+const Layout = ({ children }: LayoutProps) => {
+  return (
+    <>
+      <Navbar />
+      {children}
+      <Footer />
+    </>
+  );
+};
+
+export default Layout;
```

Comment:
This component should be moved to a file called Layout.tsx in addition to that you can instead use an index.ts file to export the Layout component 

`export { Layout } from './Layout.tsx `

this makes importing alot easier especially if the component ever grows to the point where youre exporting multiple components

Reply:
Personal preference but yes we can use capital letter for components and folders as it is more common.

Reply:
> Personal preference but yes we can use capital letter for components and folders as it is more common.

Yes but it's also a standardized practice that the majority of projects follow.